app/config.template.json

```diff
@@ -23,12 +23,12 @@
     },
     "SqlSettings": {
         "DriverName": "postgres",
-        "DataSource": "postgres://MM_USERNAME:MM_PASSWORD@DB_HOST:DB_PORT/MM_DBNAME?sslmode=disable&connect_timeout=10",
+        "DataSource": "postgres://mattermost:POSTGRES_PASSWORD@db:5432/mattermost?sslmode=disable&connect_timeout=10",
```

Comment:
The port for instance is deprecated: https://docs.docker.com/compose/link-env-deprecated/

But, ok, I understand that some people might want to configure it.
The main reason is to simplify, but if you prefer to make it configurable, I have no strong opinions against.